(feat): Agent Control (#19943)
newrelic/docs-website: src/content/docs/new-relic-control/fleet-control/authentication.mdx

---
title: "Authentication"
metaDescription: "Overview of the Agent Control authentication process."
freshnessValidatedDate: never
---
<Callout title="preview">
  We're still working on this feature, but we'd love for you to try it out!

  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
</Callout>

New Relic Fleet Control requires the use of system identities, which are non-human identities used to authenticate
and establish trust between services and applications.

System identities have two different ways to authenticate. The two supported authentication flows are:
* Private key signed JWT
* Client credentials

> During the guided installation process, the first system identity is created using client credentials, which are included
>  in the Helm chart's values. This system identity has a TTL of 12 hours. When it expires, the Agent Control Helm chart
>  deployment will fail to authenticate with the Fleet Control service, resulting in the following error:

Reviewer: ```suggestion
During the Agent Control guided installation process, the first system identity is created using client credentials, which are included in the Helm chart's values. The credentials for this system identity expire after 12 hours. When they expire, the Agent Control Helm chart deployment will fail to authenticate with the Fleet Control service, resulting in the following error:
```
